code/infrastructure-api/src/stacks/stacks.js

```diff
@@ -0,0 +1,20 @@
+import find from 'lodash/find';
```

Comment:
Seems that we may have a few mixed imports in DataLabs. Normally we have used `import { find } from 'lodash'`, but I have seen another import using / in `errorHandlers,js`

Reply:
OK, will merge this and submit another PR with a find replace.